backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-creating-backup-cr.adoc

```diff
@@ -11,7 +11,7 @@ include::_attributes/attributes-openshift-dedicated.adoc[]
 
 toc::[]
 
-You back up Kubernetes images, internal images, and persistent volumes (PVs) by creating a `Backup` custom resource (CR).
+You back up Kubernetes resources, internal images, and persistent volumes (PVs) by creating a `Backup` custom resource (CR).
```

Comment:
Rewrite as: "To back up Kubernetes resources, internal images, and persistent volumes (PVs), create a `Backup` custom resource (CR)."

Reply:
Done. thank you.